prereise/gather/griddata/transmission/geometry.py

```diff
@@ -0,0 +1,273 @@
+from dataclasses import dataclass
+from math import exp, log, pi, sqrt
+from statistics import geometric_mean
+
+from prereise.gather.griddata.transmission.const import (
+    epsilon_0,
+    mu_0,
+    relative_permeability,
+    resistivity,
+)
+
+
+@dataclass
+class Conductor:
+    """Represent a single conductor (which may be a stranded compsite).
+
+    :param float radius: outer radius of conductor.
+    :param str material: material of conductor. Used to calculate ``resistance_per_km``
+        and ``gmr`` if these aren't passed to the constructor, unnecessary otherwise.
+    :param float resistance_per_km: outer radius of conductor. Will be estimated from
```

Comment:
`outer radius of conductor` -> `resistance per kilometer`?

Reply:
Done. I guess I got pretty sloppy with my docstrings on this one.